apps/web/app/(roles)/vagas/page.tsx

```diff
@@ -59,10 +58,26 @@ async function getSkills() {
   return skills
 }
 
+async function getCountries() {
+  const { data: countries } = await supabase
+    .from('vw_countries_in_roles')
```

Comment:
where is the code for this view?

Reply:
CREATE VIEW vw_countries_in_roles AS
SELECT DISTINCT r.country
FROM "Roles" r
WHERE r.country IS NOT NULL;

Reply:
Could you please use the query instead of the view? This way we keep everything related to the application in the source code

Reply:
const { data, error } = await supabase
    .from('Roles')
    .select('country')
    .not('country', 'is', null)
    .order('country')
    
I could do it this way, but it presents a performance problem, this query would go through the entire 'Roles' table. Do you have any ideas on how to improve this query? Or should we leave it like this and implement Redis for caching?

Reply:
That is not a problem, just ensure that `country` property has an **index** to make the query faster.

Reply:
Actually, I just noticed that the supabase CLI doesn't have a distinct... Let's use the view then, just make sure to create a folder called `views` on the supabase project.